spec/lib/sequel_rails/storage_spec.rb

```diff
@@ -12,10 +12,9 @@
       },
       'test' => {
         'adapter' => 'postgres',
+        'url' => 'postgres://127.0.0.1/sequel_rails_test_storage_test',
```

Comment:
It was a bit hard for me to find a good place to add the `Storage::Abstract` tests - let me know where you think that'd be appropriate.